pennylane/ops/identity.py

```diff
@@ -177,3 +177,106 @@ def adjoint(self):
 
     def pow(self, _):
         return [Identity(wires=self.wires)]
+
+
+class GlobalPhase(Operation):
+    r"""A global phase operation that multiplies the all components of the state by :math:`\exp{-1j \phi}`.
+
+    Args:
+        phi (TensorLike): the global phase
+        wires (Iterable[Any] or Any): Wire label(s) that the global phase acts on.
+
```

Comment:
Not sure if this is still WIP but please could we do a polish of the docstrings - the docs for a new feature are as important as the code itself.

We can:
- Add an example section (see [here](https://xanaduai-pennylane--4471.com.readthedocs.build/en/4471/code/api/pennylane.PCPhase.html) for example)
- Tidy up the `\exp{-1j \phi}` (it doesn't look nice in the built docs)
- Is there an `id` argument?
- Tidy up "multiplies the all components"
- What about the "Details:" section common in other ops?

Reply:
Thanks, looks grreat!